db_assessment/optimusprime.py

```diff
@@ -237,11 +264,15 @@ def argumentsParser():
     
     parser.add_argument("-consolidatedataframes", default=False, help="Consolidate CSV files before importing.", action="store_true")
 
+    parser.add_argument("-remote", default=False, help="Leverage remote API", action="store_true")
+
+    parser.add_argument("-remoteurl", type=str, default="https://op-api-3qhhvv7zvq-uc.a.run.app", help="Leverage remote API")
+    
 
     # Consolidates different collection IDs found in the OS (dbResults/*log) into a single CSV per file type. 
     # For example: dbResults has 52 files. Meaning, 2 collection IDs (each one has 26 different file types). 
     # After the consolidation it produces 26 *consolidatedlogs.log which would have data from both collection IDs 
-    parser.add_argument("-cl", "--consolidatelogs", default=False, help="consolidate all CSV files opdb*log found in dbResults/ directory", action="store_true")
```

Comment:
Hey Abel, quick question..

We are removing the "action" attribute. Is there any reason? I do not remember the default value for that maybe it is the same.

Reply:
Good catch, i dont think this was intended to get into the changeset

Reply:
Sure thing. I am glad to help. :-)

Reply:
done!